pkg/db/task.go

```diff
@@ -0,0 +1,110 @@
+package db
+
+import (
+	"fmt"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func AddTask(task *Task) (int64, error) {
+	query := `INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)`
+	res, err := db.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+
+	if err != nil {
+		return 0, fmt.Errorf("ошибка при добавлении задачи: %v", err)
+	}
+
+	return res.LastInsertId()
+}
+
+func Tasks(limit int) ([]*Task, error) {
+	rows, err := db.Query("SELECT id, date, title, comment, repeat FROM scheduler ORDER BY date ASC LIMIT ?", limit)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var tasks []*Task
+
+	for rows.Next() {
+		task := &Task{}
+		err := rows.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+		if err != nil {
+			return nil, err
```

Comment:
Лучше все ошибки оборачивать
```suggestion
			return nil, fmt.Errorf("ошибка при сканировании задачи: %w", err)
```